src/main/java/frc/trigon/robot/subsystems/swerve/Swerve.java

```diff
@@ -0,0 +1,81 @@
+package frc.trigon.robot.subsystems.swerve;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Swerve extends SubsystemBase {
+    private final static Swerve INSTANCE = new Swerve();
+
+    public Swerve() {
+        zeroHeading();
+    }
+
+    public static Swerve getInstance() {
+        return INSTANCE;
+    }
+
+    /**
+     * Drives the swerve relative to itself.
```

Comment:
This function is confusing to someone new.
You can say "Drives the swerve with the given velocities, relative to the robot"